source/docs/3/origin_destination_matrix.rst

```diff
@@ -90,115 +94,57 @@ Procedure
   .. image:: /static/3/origin_destination_matrix/images/8.png
     :align: center
   
-9. This algorithm helps find the distances along the network between selected origin and destination layers. Select ``Street_Centerlines`` as the :guilabel:`Network layer`. Select ``origin_points`` as the :guilabel:`From-Points layer` and ``OBJECTID`` as the :guilabel:`Unique Point ID field`. Similarly, set ``destination_points`` as the :guilabel:`To-Points Layer` and ``OBJECTID`` as  the :guilabel:`Unique Point ID field`. Set the :guilabel:`Optimization Criterion` as ``Shortest Path``.
+9. This algorithm helps find the distances along with the network between selected origin and destination layers. Select ``Roadway_Block`` as the :guilabel:`Network layer`. Select ``origin_points`` as the :guilabel:`From-Points layer` and ``OBJECTID`` as the :guilabel:`Unique Point ID field`. Similarly, set ``destination_points`` as the :guilabel:`To-Points Layer` and ``OBJECTID`` as  the :guilabel:`Unique Point ID field`. Set the :guilabel:`Optimization Criterion` as ``Shortest Path (distance optimization)``.
```

Comment:
Need to be OBJECTID_1, fix both text , screenshot and subsequent resulrs.

Reply:
updated.